pyproject.toml

```diff
@@ -14,14 +14,23 @@ classifiers=[
 dynamic = ["version"]
 dependencies = [
   'click>=8.0',
-  'ujson>=5.4.0',
-  'requests',
-  'psycopg2-binary>=2.9.3',
   'countryinfo>=0.1.2',
+  'lb_matching_tools==v2024.01.30.1',
   'liblistenbrainz>=v0.5.5',
+  'more_itertools',
+  'mutagen==1.46.0',
+  'nmslib@git+https://github.com/nmslib/nmslib.git@master',
+  'peewee==3.17.0',
+  'psycopg2-binary>=2.9.3', 
+  'py-sonic==1.0.0',
   'python-dateutil>=2.8.2',
+  'regex==2023.6.3',
+  'requests>=2.31.0',
+  'scikit-learn==1.2.1',
   'spotipy>=2.22.1',
-  'more_itertools'
+  'tqdm',
+  'ujson>=5.4.0',
+  'Unidecode==1.3.6',
```

Comment:
Since this is a library we probably should not exactly pin dependencies, but define a minimum version.

Reply:
Done.